packages/go_router/lib/fix_data.yaml

```
@@ -21,7 +21,7 @@ transforms:
     date: 2023-04-24
     bulkApply: true
     element:
-      uris: [ 'go_router.dart' ]
+      uris: [ 'go_router.dart', 'package:go_router/go_router.dart' ]
```

Let's add a todo comment to mention this is a workaround for https://github.com/dart-lang/sdk/issues/52233

---

is this a good syntax ?
```suggestion
      # TODO: Workaround for https://github.com/dart-lang/sdk/issues/52233
      uris: [ 'go_router.dart', 'package:go_router/go_router.dart' ]
```

---

something like this
```
TODO(ahmednfwela): Workaround for https://github.com/dart-lang/sdk/issues/52233
```

adding your name doesn't mean you have to clean up, it is just a reference